pkg/smartcontract/manifest/standard/nep24.go

```diff
@@ -0,0 +1,47 @@
+package standard
+
+import (
+	"github.com/nspcc-dev/neo-go/pkg/smartcontract"
+	"github.com/nspcc-dev/neo-go/pkg/smartcontract/manifest"
+)
+
+// Nep24 is a NEP-24 Standard for NFT royalties.
+var Nep24 = &Standard{
+	Manifest: manifest.Manifest{
+		ABI: manifest.ABI{
+			Methods: []manifest.Method{
+				{
+					Name: "royaltyInfo",
+					Parameters: []manifest.Parameter{
+						{Name: "tokenId", Type: smartcontract.ByteArrayType},
+						{Name: "royaltyToken", Type: smartcontract.Hash160Type},
+						{Name: "salePrice", Type: smartcontract.IntegerType},
+					},
+					ReturnType: smartcontract.ArrayType,
+					Safe:       true,
+				},
+			},
+		},
+	},
+	Required: []string{manifest.NEP11StandardName},
+}
+
+// RoyaltiesTransfer an event that MUST be emitted by marketplaces that support the `royaltyInfo` method.
```

Comment:
It's not true. Marketplaces don't support this method, this method may be supported by NEP11 token contract. If you're copy-pasting from the standard, then make it properly, here's what the standard tells about it:
> MUST trigger after marketplaces transferring royalties to the royalty recipient if royaltyInfo method is implemented.

Reply:
The standard also tells:

> The royaltyInfo() function MUST NOT ignore the salePrice, royaltyAmount MAY be dynamic due to salePrice and time.
> 
> Marketplaces that support this standard MUST emit the event, RoyaltiesTransferred for each recipient, after sending a payment.

Which is literally my comment.  So  the comment `// RoyaltiesTransfer is an event that MUST trigger after marketplaces transferring royalties to the royalty recipient if royaltyInfo method is implemented.` is more correct?

Reply:
Oh, I should have left a comment to the standard when it was on the review stage. OK, then choose either you'd like.

> an event that MUST trigger

Grammar: an event that MUST be triggered